pcdsdevices/signal.py

```diff
@@ -811,8 +814,214 @@ class FakeEpicsSignalROEditMD(FakeEpicsSignalRO):
     Add to this if you need it to actually work for your test.
     """
     def _override_metadata(self, **kwargs):
-        pass
+        ...
 
 
 fake_device_cache[EpicsSignalEditMD] = FakeEpicsSignalEditMD
 fake_device_cache[EpicsSignalROEditMD] = FakeEpicsSignalROEditMD
+
+
+class EpicsSignalBaseEditEnum(EpicsSignalBaseEditMD):
+    """
+    EpicsSignal variant which allows for user correction of enum strings.
+
+    Parameters
+    ----------
+    enum_attrs : list of str
+        List of attribute names, relative to the parent device.  That is to
```

Comment:
Some clarification that these are not general attribute strings, these are signals that hold string values
```suggestion
        List of signal names, relative to the parent device.  That is to
```